lib/internal/bootstrap/loaders.js

```diff
@@ -270,29 +259,29 @@
       const cache = codeCacheHash[id] &&
         (codeCacheHash[id] === sourceHash[id]) ? codeCache[id] : undefined;
 
-      // (code, filename, lineOffset, columnOffset
-      // cachedData, produceCachedData, parsingContext)
-      const script = new ContextifyScript(
-        source, this.filename, 0, 0,
-        cache, false, undefined
+      const fn = compileFunction(
+        source,
+        this.filename,
+        0,
+        0,
+        cache,
+        false,
+        undefined,
+        [],
```

Comment:
☝️ I think `contextExtensions` can be `undefined` too.

Reply:
@jdalton I used an empty array since that's the default value I used in `vm.compileFunction` and wanted to make this as consistent as I could. I just rechecked the code to make sure, the two *should* be functionally similar, resulting in an empty `std::vector<Local<Object>> context_extensions`.

Reply:
My point was more reducing throw away object creation.